src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.JsonTranscoding/Internal/Json/EnumConverter.cs

```diff
@@ -26,7 +26,13 @@ public EnumConverter(JsonContext context) : base(context)
                 {
                     throw new InvalidOperationException($"Unable to resolve descriptor for {typeToConvert}.");
                 }
-                var valueDescriptor = enumDescriptor.FindValueByName(reader.GetString()!);
+
+                var value = reader.GetString()!;
+                var valueDescriptor = enumDescriptor.FindValueByName(value);
+                if (valueDescriptor == null)
+                {
+                    throw new InvalidOperationException($"Invalid enum value: {value} for enum type: {typeToConvert}.");
```

Comment:
I used this error message to be simillar as used in [NullValueConverter](https://github.com/dotnet/aspnetcore/blob/b4b43e39bc3b79b773d81859efd0f4ad558c24e7/src/Grpc/JsonTranscoding/src/Microsoft.AspNetCore.Grpc.JsonTranscoding/Internal/Json/NullValueConverter.cs#L26).
What do you think about keep it the same?

Reply:
I copied that message from Google.Protobuf and its JSON code. I didn’t improve it because there was a lot of other thing to think about at the time. I like the new error message more because it’s more idiomatic to other .NET exception messages.

You could change the other one now or I’ll do it sometime in the future.

Reply:
Applying changes to null converter breaks test `NullValue_NonDefaultValue_String` because old deserialized throws exception with old message and it looks it requires more changes or just disable test for old deserialized, so I changed only enum converter